README.md

```diff
@@ -31,9 +31,9 @@ The `ilab` tool allows you to interact with the IBM AI model `Merlinite` or `Gra
 
 > **Note:** Before proceeding, it might be beneficial to check out the [Contributing](https://github.com/instruct-lab/community/blob/main/CONTRIBUTING.md) guide for an overview of contributing practices and expectations. Additionally, you should consider joining the [InstructLab community Slack channel](https://github.com/instructlab/community/blob/main/InstructLab_SLACK_GUIDE.md).
 
-1. Navigate to the `ilab` CLI repository and follow the instructions in the [README.md](https://github.com/instructlab/instructlab/blob/main/README.md). The README.md instructs you on how to perform the following:
+1. Navigate to the `ilab` repository and follow the instructions in the [README.md](https://github.com/instructlab/instructlab/blob/main/README.md). The README.md instructs you on how to perform the following:
```

Comment:
It is not really the ilab repository since it is named instructlab. Maybe you need to say "`instructlab` repository". Same comment for the remaining uses of "`ilab` repository".